backend/core/settings/prod.py

```
@@ -0,0 +1,63 @@
+DEFAULT_ALLOWED_HOSTS = ["apatie.example"]
+ALLOWED_HOSTS = ENV.list("ALLOWED_HOSTS", default=DEFAULT_ALLOWED_HOSTS)
+if not ALLOWED_HOSTS or ALLOWED_HOSTS == ["*"]:
+    raise ImproperlyConfigured(
+        "ALLOWED_HOSTS must be configured with concrete domains in production."
+    )
+
+
+# Security hardening recommended for public deployments.
+SECURE_SSL_REDIRECT = True
+SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+SECURE_HSTS_SECONDS = ENV.int("SECURE_HSTS_SECONDS", default=31536000)
```

Ruff (0.13.3)

`ENV` may be undefined, or defined from star imports

(F405)